Documentation/capabilities-guide.md

```diff
@@ -222,6 +223,101 @@ root
 nc: bind: Permission denied
 ```
 
+## Overriding capabilities
+
+Capabilities sets are typically defined when creating images, as they are tightly
+linked to specific app requirements. However, image consumers may need to further
+tweak/restrict the set of available capabilities in specific local scenarios.
+This can be done either by permanently patching the manifest of specific images, or
+by overriding capability isolators via command line arguments.
+
+### Patching images
+
+Image manifests can be manipulated manually, by unpacking the image and editing
+the manifest file, or via helper tools like `actool`.
+In order to override pre-defined capabilities set, it is enough to replace any
```

Comment:
"To override an image's pre-defined capabilities set, replace the existing capabilities isolators in the image with new isolators defining the desired capabilities."